Web Service plan is not common for users.

kemura opened the pull request:
Hi, I got a question from customers saying "Is Web service plan equal to App Service Plan". 
I think they are right but the document is a little bit confusing. So I suggest to change App Service plan

azurecla commented:
Hi __@wesmc7777__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Commit:
Web Service plan is not common for users.

Hi, I got a question from customers saying "Is Web service plan equal to App Service Plan". 
I think they are right but the document is a little bit confusing. So I suggest to change App Service plan

Commit:
Merge branch 'master' into patch-1
